### DISCUSSION TITLE
A shortcut to get back to the library

### DISCUSSION THREAD

**Discussion by maruf-ahmed-bhuiyan** [johnfactotum/foliate#656] [Ideas]
Sometimes when I am reading an epub, I would like to go back to the library quickly. I can't find a shortcut to get back to the library. I think this shortcut would be really handy since the current solutions require you to click the menu button and clicking the library which is very cumbersome. What do you guys think?

**Comment by johnfactotum** [comment]
Yeah, I think that'd be a good idea.

**Comment by itprojects** [comment]
What would be your preferred shortcut for such an action?

Is there such a shortcut key combination already in Foliate?

How do other GNOME apps do this?

**Reply by maruf-ahmed-bhuiyan** [reply]
I think Ctrl + L would be a good shortcut.

**Reply by johnfactotum** [reply]
Ctrl + L is already used for the location popover.

**Reply by itprojects** [reply]
Shift + L?
Alt + L?

**Comment by ernesto1** [comment]
I want to add in my support for this feature.  As of v.2.6.1 the only way to open a new file is going back to the library.
Since the event is so common in a reading flow, it warrants a button on the title bar to perform the action quickly.

The need for this becomes more apparent when reading multiple short pieces such as comics (.cbz) in a single sitting.

**Comment by JohannPopper** [comment]
I wouldn't mind a keyboard shortcut personally, but with an eye toward Foliate being accepted into GNOME Circle as well as consistent GNOME app usability standards (ref: GNOME Visual layout documentation), GNOME app guidelines typically encourage GUI solutions for primary functions integrated into the header bar hierarchically: " you should place dominant controls above and to the left of the controls and content that they affect. Header bars are a key design pattern in this respect."

Quick access to book management is a primary function than would logically reside at a level above opened content, but of course secondary to the table of contents for the currently opened book. Therefore, I would suggest a Library button with something like an open book symbolic icon situated in the upper left of the header bar, immediately to the right of the current contents button. 

IMHO.

**Reply by johnfactotum** [reply]
I do plan on adding a library button, similar to other GNOME apps (or indeed most e-book apps). But I'll add it as part of merging the library view and the book view into a single window. See #509.
